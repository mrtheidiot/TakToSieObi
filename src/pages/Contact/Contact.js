import Banner from "../../components/Banner/Banner";

import classes from "./Contact.module.css";

const Contact = () => {
  return (
    <>
      <Banner id={2} />
      <div className={classes.wrapper} data-aos="fade-in" data-testid="contact">
        <section className={classes.company_data}>
          <h3>Dane firmy:</h3>
          <p>Tak To Się Obi Dorota Papadopoulos</p>
          <p>ul. Gwiazdowa 36</p>
          <p>70-554 Szczecin</p>
        </section>
        <section className={classes.transfer_data}>
          <h3>Dane do przelewu:</h3>
          <p>Tak To Się Obi Dorota Papadopoulos</p>
          <p>ING BANK Śląski:</p>
          <p>68 1050 1559 1000 0092 6329 1545</p>
        </section>
        <section className={classes.contact_data}>
          <h3>Dane kontaktowe:</h3>
          <p>Tel: 697 288 123</p>
          <p>ul. Gwiazdowa 36</p>
          <p>email: taktosieobi@gmail.com</p>
        </section>
        <section className={classes.obi_links}>
          <a
            href="https://facebook.com/ObiTraining"
            target="_blank"
            rel="noreferrer noopener"
          >
            <i className="fab fa-2x fa-facebook fejs"></i>ObiTraining
          </a>

          <a
            href="https://www.instagram.com/taktosieobi"
            target="_blank"
            rel="noreferrer noopener"
          >
            <i className="fab fa-2x fa-instagram insta"></i>taktosieobi
          </a>
        </section>
        <section className={classes.husse_links}>
          <a
            href="https://www.facebook.com/AnimalevraHusse"
            target="_blank"
            rel="noreferrer noopener"
          >
            <i className="fab fa-2x fa-facebook fejs"></i> AnimalevraHusse
          </a>

          <a
            href="https://www.instagram.com/husse_szczecin"
            target="_blank"
            rel="noreferrer noopener"
          >
            <i className="fab fa-2x fa-instagram insta"></i> husse_szczecin
          </a>
        </section>
        <section className={classes.messenger}>
          <a
            href="https://m.me/ObiTrening"
            target="_blank"
            rel="noreferrer noopener"
          >
            <i className="fab fa-2x fa-facebook-messenger"></i>
          </a>
          Messenger Me!
        </section>
      </div>
    </>
  );
};

export default Contact;

// import classes from "./kontakt.module.css";
// import './Kontakt.css';

// const Kontakt = () => {
//   return (
//     <div className={classes.wrapper}>
//       <div className={classes.seg1}>
//         <div className={classes.title}>Dane firmy:</div>
//         <div>
//           Animalevra Dorota Alewras
//           <br></br>
//           ul. Monte Cassino 36/7
//           <br></br> 70-465 Szczecin
//         </div>
//         <div>Dane do przelewu:</div>
//         <div>Animalevra Dorota Alewras</div>
//         <div>
//           ING BANK Śląski:
//           <br></br>
//           68 1050 1559 1000 0092 6329 1545
//         </div>
//       </div>
//       <div className={classes.seg2}>
//         <div className={classes.title}>Dane kontaktowe:</div>
//         <div>
//           Tel: 697 288 234
//           <br></br>
//           email: taktosieobi@gmail.com
//           <br></br>
//           <div className={classes.messenger}>
//             <a
//               href="https://m.me/ObiTrening"
//               target="_blank"
//               rel="noreferrer noopener"
//             >
//               <i className="fab fa-2x fa-facebook-messenger fejs"></i>{" "}
//             </a>
//             Messenger Me!
//           </div>
//         </div>
//       </div>
//       <div className={classes.seg3}>
//         <div className={classes.socialwrapper}>
//           <div className={classes.title}>Obserwuj nas tu:</div>
//           <div className={classes.socialmedia}>
// <a
//     href="https://facebook.com/ObiTraining"
//     target="_blank"
//     rel="noreferrer noopener"
//   >
//   <i className="fab fa-2x fa-facebook fejs"></i> </a>ObiTraining

//           </div>
//           <div className={classes.socialmedia}>
// <a
//     href="https://www.instagram.com/taktosieobi"
//     target="_blank"
//     rel="noreferrer noopener"
//   >

// <i className="fab fa-2x fa-instagram insta"></i></a> taktosieobi

//           </div>
//         </div>
//         <div className={classes.socialwrapper}>
//           <div className={classes.title}>Karma Husse:</div>
//           <div className={classes.socialmedia}>
// <a
//     href="https://www.facebook.com/AnimalevraHusse"
//     target="_blank"
//     rel="noreferrer noopener"
//   >
//   <i className="fab fa-2x fa-facebook fejs"></i></a> AnimalevraHusse

// </div>
// <div className={classes.socialmedia}>
// <a
//     href="https://www.instagram.com/husse_szczecin"
//     target="_blank"
//     rel="noreferrer noopener"
//   >
// <i class="fab fa-2x fa-instagram insta"></i></a> husse_szczecin

//           </div>
//         </div>
//       </div>
//     </div>
//   );
// };

// export default Kontakt;
