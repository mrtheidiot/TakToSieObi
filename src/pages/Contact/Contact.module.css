.wrapper {
    width: 50%;
    margin: 30px auto;
    display: grid;
    grid-template-columns: 1fr 1fr;
    grid-template-rows: 1fr 1fr 1fr;
    background-color: bisque;
    padding: 15px;
    border-radius: 10px;
    align-content: space-around;
    align-items: center;
    justify-items: center;
}

.wrapper section {
    width: 100%;
}

.company_data {
    grid-column: 1 / 2;
    grid-row: 1 / 2;
}

.transfer_data {
    grid-column: 1 / 2;
    grid-row: 2 / 3;
}

.contact_data {
    grid-column: 1 / 2; 
    grid-row: 3 / 4;
}

.messenger {
    grid-column: 2 / 3;
    font-size: 1.2rem;
    display: flex;
    align-items: center;
}

.obi_links {
    grid-column: 2 / 3;
    display: flex;
    font-size: 1.2rem;
    flex-direction: column;
}

.husse_links {
    grid-column: 2 / 3;
    font-size: 1.2rem;
    display: flex;
    flex-direction: column;
}

.wrapper a {
    text-decoration: none;
    display: flex;
    align-items: center;
    
}

i {
    margin: 5px 10px 5px 0;
}

@media (max-width: 868px) {
    .wrapper {
        grid-template-columns: 1fr;
    }
    
    .company_data {
        grid-column: 1 / 2;
    }
    
    .transfer_data {
        grid-column: 1 / 2;
    }
    
    .contact_data {
        grid-column: 1 / 2; 
    }
    .obi_links {
        grid-column: 1 / 2;
        margin-top: 20px;
    }
    .husse_links {
        grid-column: 1 / 2;
    }
    
    .messenger {
        grid-column: 1 / 2;
        font-size: 1.2rem;
    }
}