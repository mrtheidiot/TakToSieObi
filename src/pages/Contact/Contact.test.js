import { render } from "@testing-library/react";
import { BrowserRouter as Router } from "react-router-dom";
import "@testing-library/jest-dom";

import Contact from "./Contact";

it("checks if the Contact Page matches snapshot", () => {
  const {
    container: { firstChild },
  } = render(
    <Router>
      <Contact />
    </Router>
  );

  expect(firstChild).toMatchSnapshot();
});
