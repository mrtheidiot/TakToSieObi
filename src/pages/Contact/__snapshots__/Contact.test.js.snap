// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`checks if the Contact Page matches snapshot 1`] = `
<div
  class="banner"
  style="position: relative; overflow: hidden; width: 100%;"
>
  <div
    data-testid="layer-0"
    style="background-image: url(banner2.png); background-position: center; background-size: cover; position: absolute; top: -150px; left: 0px; right: 0px; bottom: -150px; will-change: transform; transform: translateY(150px);"
  />
</div>
`;
